core/osrd-reporting/src/main/java/fr/sncf/osrd/reporting/Reflection.java

```diff
@@ -15,8 +15,6 @@
 import java.util.List;
 
 public class Reflection {
```

Comment:
It's kind a weird for a reflection helper to end up in a package called "reporting" don't you think?

Reply:
This reflection helper is only used in the classes of the `reporting` package so I grouped them